docs/dev/table/connectors/jdbc.md

```diff
@@ -0,0 +1,395 @@
+      <td>Integer</td>
+      <td>The max number of rows of lookup cache. "lookup.cache.max-rows" and "lookup.cache.ttl" options must all be specified if any of them is specified.</td>
+    </tr>
+    <tr>
+      <td><h5>lookup.cache.ttl</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>Integer</td>
+      <td>The max time to live for each rows in lookup cache.</td>
+    </tr>
+    <tr>
+      <td><h5>lookup.max-retries</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>Integer</td>
+      <td>The max retry times if lookup database failed.</td>
+    </tr>
+    <tr>
+      <td><h5>sink.buffer-flush.max-rows</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">5000</td>
+      <td>Integer</td>
+      <td>The max size of buffered records before flush.</td>
+    </tr> 
+    <tr>
+      <td><h5>sink.buffer-flush.interval</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>Duration</td>
+      <td>The flush interval mills, over this time, asynchronous threads will flush data.</td>
+    </tr>
+    <tr>
+      <td><h5>sink.max-retries</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">3</td>
+      <td>Integer</td>
+      <td>The max retry times if writing records to database failed.</td>
+    </tr>          
+    </tbody>
+</table>
+
+Data Type Mapping
+----------------
+Flink support connect to several JDBC dialect，the field type mappings between Flink and current supported databases are as follows:
+
+<table class="table table-bordered">
+    <thead>
+      <tr>
+        <th class="text-left">Flink Data Type</th>
+        <th class="text-left">MySQL Type</th>
+        <th class="text-left">PostgreSQL Type</th>
+        <th class="text-left">Derby Type</th>
+      </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td>BOOLEAN</td>
+      <td>TINYINT</td>
+      <td>BOOLEAN</td>
+      <td>BOOLEAN</td>
+    </tr>
+    <tr>
+      <td>TINYINT</td>
+      <td>TINYINT</td>
+      <td>SMALLINT</td>
+      <td>SMALLINT</td>
+    </tr>
+    <tr>
+      <td>SMALLINT</td>
+      <td>SMALLINT</td>
+      <td>SMALLINT</td>
+      <td>SMALLINT</td>
+    </tr>
+    <tr>
+      <td>INT</td>
+      <td>INT</td>
+      <td>INT</td>
+      <td>INT</td>
+    </tr>
+    <tr>
+      <td>BIGINT</td>
+      <td>BIGINT</td>
+      <td>BIGINT</td>
+      <td>BIGINT</td>
+    </tr>
+    <tr>
+      <td>FLOAT</td>
+      <td>FLOAT</td>
+      <td>REAL</td>
+      <td>FLOAT</td>
+    </tr>
+    <tr>
+      <td>DOUBLE</td>
+      <td>DOUBLE</td>
+      <td>DOUBLE PRECISION</td>
+      <td>DOUBLE PRECISION</td>
+    </tr>
+    <tr>
+      <td>DECIMAL</td>
+      <td>DECIMAL</td>
+      <td>DECIMAL</td>
+      <td>DECIMAL</td>
+    </tr>
+    <tr>
+      <td>CHAR</td>
+      <td>VARCHAR</td>
+      <td>VARCHAR</td>
+      <td>VARCHAR</td>
```

Comment:
Not sure about this .

Reply:
The data type mapping is not fully aligned with this FILP: https://cwiki.apache.org/confluence/display/FLINK/FLIP-93%3A+JDBC+catalog+and+Postgres+catalog